test/indent/rust/after-where/in

```diff
@@ -1,5 +1,3 @@
-    impl X for T where%( )
```

Comment:
why did this test case go ?

Reply:
It is not important, also `rustfmt` does not forward with `where` at the back IIRC.